File: TutorialPractice/Questions/Question2.cpp
#include <iostream>
using namespace std;

int main() {
    int n;

    cout << "Enter the size of the array: ";
    cin >> n;

    int arr[n];  
    cout << "Enter the elements of the array:" << endl;
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }

    cout << "The even values of the arrays are:" << endl;
    for (int i = 0; i < n; i++) {
        if(arr[i]%2 == 0){
        cout << arr[i] << " ";

        }
    }

    return 0;
}

File: TutorialPractice/Questions/Question2.js
const prompt = require('prompt-sync')();
let n=prompt("Enter the size of the array: ")
n=parseInt(n);
let arr=[];
console.log("Enter the elements of the array: ");
for(let i=0; i<n; i++){
    arr[i]=parseInt(prompt(`Enter element ${i+1}: `));
}
console.log("The even values of the arrays are: ");
for(let i=0; i<arr.length; i++){
    if(arr[i]%2==0){
        console.log(arr[i]);
    }
}